tests/testlibrary.c

```diff
@@ -274,7 +290,7 @@ test_languages_config (void)
   g_assert_no_error (error);
   g_assert_cmpstr (value[0], ==, "en");
```

Comment:
Not a bug introduced by this PR but I think there's a missing `g_assert_null (value[1])` here.